### PR TITLE
Doc for smart lock and web credentials!

### PR DESCRIPTION
**Pull request by joawan**

### CODE REVIEW

File: resources/articles/shared-web-credentials-and-google-smart-lock.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+:frontpage
+:category api-integration
+:body
+
+Shared Web Credentials for iOS and Google Smart Lock for Android will simplify the login step for all users using native apps.
+User having enabled these features, will be prompted in enabled apps to choose which credentials to use, and they will be automatically filled in.
+
+### Configuration
+
+Shared Web Credentials and Google Smart Lock can be configured in SelfService, under Basic settings.
```
Comment (Prazmok) [inline resources/articles/shared-web-credentials-and-google-smart-lock.md]: Maybe you could link SelfService to [http://localhost:3000/selfservice/access/](http://localhost:3000/selfservice/access/)?

**Comment by Prazmok** [comment]
LGTM. (Beyond my suggestion about SelfService everything looks OK)

File: resources/articles/shared-web-credentials-and-google-smart-lock.md (hunk 1 above)
Comment (joawan) [inline resources/articles/shared-web-credentials-and-google-smart-lock.md]: A link to what?

Comment (joawan) [inline resources/articles/shared-web-credentials-and-google-smart-lock.md]: ah, now I got you. http://techdocs.spid.no/selfservice/access/
I'll add that

**Pull request merged**